src/styles/components/form-elements/styles.less

```diff
@@ -35,9 +41,20 @@
     position: absolute;
   }
 
-  .form-remove {
+  .form-group-container-action-button {
```

Comment:
Ah, ok. But it is not on the action button? It's a wrapper around the action button, right?